core/src/test/scala/integration/kafka/api/SaslSetup.scala

```diff
@@ -131,7 +131,7 @@ trait SaslSetup {
 
   def kafkaServerSaslProperties(serverSaslMechanisms: Seq[String], interBrokerSaslMechanism: String): Properties = {
     val props = new Properties
-    props.put(KafkaConfig.SaslMechanismInterBrokerProtocolProp, interBrokerSaslMechanism)
+    props.put(KafkaSecurityConfigs.SASL_MECHANISM_INTER_BROKER_PROTOCOL_CONFIG, interBrokerSaslMechanism)
```

Comment:
ok, that is why I'm not fan of `KafkaConfig`. It makes us mix them up. Maybe we can use `BrokerSecurityConfigs` as much as possible, and leave `KafkaSecurityConfigs` to be a kind of "index" for human developers?